src/System.Management.Automation/engine/TypeTable.cs

```diff
@@ -3549,20 +3549,22 @@ internal Collection<String> GetSpecificProperties(ConsolidatedString types)
             return PSObject.TransformMemberInfoCollection<PSMemberInfo, T>(GetMembers(types));
         }
 
-        internal PSMemberInfoInternalCollection<PSMemberInfo> GetMembers(ConsolidatedString types)
+        internal PSMemberInfoInternalCollection<PSMemberInfo> GetMembers(ConsolidatedString types, bool inObjectBuilderScope = false)
```

Comment:
The name is very confusing me - in the method we use GetOrAdd() - it is _modify_ operation in _Get_ method! I'd rename the method to GetOrAddMembers().